storage/backend/ceph/ceph_test.go

```diff
@@ -109,6 +128,20 @@ func (s *cephSuite) TestMountUnmountVolume(c *C) {
 	s.readWriteTest(c, mp)
 	c.Assert(mountDriver.Unmount(driverOpts), IsNil)
 	c.Assert(crudDriver.Destroy(driverOpts), IsNil)
+
+	driverOpts = storage.DriverOptions{
+		Volume:    volumeSpecTestPool,
+		FSOptions: filesystems["ext4"],
+		Timeout:   5 * time.Second,
+	}
+
+	if done {
+		return
+	}
+
+	done = true
+
+	goto again
```

Comment:
Does it actually perform 2 rounds of execution with different volume specification(volumeSpec, volumeSpecTestPool ? And is it why the function doesn't return on first successful completion?